### PR TITLE
Added some documentation about jax-rs applications and @OpenAPIDefinition

### PR DESCRIPTION
**Pull request by EricWittmann**

### CODE REVIEW

**Comment by EricWittmann** [comment]
@gsmet ping

File: docs/src/main/asciidoc/openapi-swaggerui.adoc [hunk 1]
```diff
@@ -250,6 +250,44 @@ quarkus.smallrye-openapi.path=/swagger
 
 Hit `CTRL+C` to stop the application.
 
+== Providing Application Level OpenAPI Annotations
+
+There are some Microprofile-OpenAPI annotations which describe global API information, such as the following:
```
Comment (gsmet) [inline docs/src/main/asciidoc/openapi-swaggerui.adoc]:
```suggestion
There are some MicroProfile OpenAPI annotations which describe global API information, such as the following:
```

**Review by gsmet — CHANGES_REQUESTED** [review]
Thanks a lot!

I added one very minor comment. Could you include it and squash everything? I'll merge then.

**Comment by EricWittmann** [comment]
Can you squash when merging?

**Comment by gsmet** [comment]
Not with a merged commit. I did it manually.

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Merged, thanks!

**Pull request merged**